src/Routes/Devices/DeviceTable.js

```diff
@@ -274,7 +274,8 @@ const DeviceTable = ({
         title: 'Update',
         onClick: (_event, _rowId, rowData) => {
           history.push({
-            pathname: `${deviceBaseUrl}/${rowData.rowInfo.id}/update`,
+            pathname: `/${rowData.rowInfo.id}/update`,
```

Comment:
The commented out code below this line should be used instead of this line, deviceBaseUrl is needed to open update device page